application.go

```diff
@@ -353,6 +353,19 @@ func (r *marathonClient) Application(name string) (*Application, error) {
 	return wrapper.Application, nil
 }
 
+// ApplicationConfiguration retrieves the application configuration from marathon
+// 		name: 		the id used to identify the application
+// 		version:  the version of the configuration you would like to receive
+func (r *marathonClient) ApplicationConfiguration(name, version string) (*Application, error) {
+	var app *Application
+
+	if err := r.apiGet(fmt.Sprintf("%s/%s/versions/%s", marathonAPIApps, trimRootPath(name), version), nil, &app); err != nil {
```

Comment:
The `apiGet()` parameter passing can be simplified by employing `buildURI()` (similar to how the [ApplicationVersions implementation](https://github.com/gambol99/go-marathon/blob/de3f7b297e825ac61224ad3a3c0b88ba814a23cb/application.go#L321) does it) like this:

```
uri := fmt.Sprintf("%s/versions/%s", buildURI(name), version)
if err := r.apiGet(uri, nil, &app); err != nil {
```